packages/esm-ohri-pmtct-app/src/views/summary-tabs/ohri-patient-tabs.scss

```diff
@@ -0,0 +1,56 @@
+@use '@carbon/styles/scss/spacing';
+@import '../../root.scss';
+
+.widgetContainer {
+  background-color: $ui-background;
+}
+
+.widgetHeaderContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: spacing.$spacing-04 0 spacing.$spacing-04 spacing.$spacing-05;
+}
+
+.widgetHeaderContainer > h4:after {
+  content: '';
+  display: block;
+  width: 2rem;
+  padding-top: 0.188rem;
+  border-bottom: 0.375rem solid $brand-color-01;
+}
+
+.toggleButtons {
+  width: fit-content;
+  margin: 0 spacing.$spacing-03;
+}
+
+.searchField {
+  width: 100%;
+  max-width: 250px;
+  border: 0px !important;
+}
+
+.searchBox {
+  width: 100%;
+  display: flex;
+  justify-content: right;
+}
+
+.container {
+  margin-top: 16px;
+  margin-left: inherit;
+  width: 100%;
+  overflow-y: hidden;
+}
+
+.tabContainer{
+margin-top: 16px;
+padding-left: 1rem;
+background-color: #ffffff;
+
+}
+
+.tabContainer li button{
+width: 100% !important;
+}
```

Comment:
not sure if github isn't displaying this correctly but this needs to be indented corrently

Reply:
@pirupius Changed